apps/sailfish/qml/components/GesturePad.qml

```diff
@@ -31,12 +31,8 @@ Item {
 
     HapticsEffect {
         id: rumbleEffect
-        attackIntensity: 0
-        attackTime: 250
-        intensity: 1.0
-        fadeTime: 250
-        fadeIntensity: 0
-        period: 100
+        intensity: 0.05
```

Comment:
the period and intensity interact in subtle ways, so increasing the period allows for the haptic hardware to spin up and increase intensity. The intensity feels more or less the same as the 0.50 one with shorter period, but more stretched out.